docs/INSTALLATION.md

```diff
@@ -0,0 +1,263 @@
+* `TUORIS_HOST` - Hostname (or IP address) + port of the [Tuoris](https://github.com/fvictor/tuoris) service (dependency of SVG application).
+
+Before installing [**OVE Asset Services**](https://github.com/ove/ove-asset-services) you must configure the environment variables:
+
+* `MYSQL_RANDOM_ROOT_PASSWORD` - This variable is mandatory and we recommend it be set to `yes`. This will generate a random initial password for the root user on the [MariaDB](https://mariadb.org/) database.
+* `MYSQL_DATABASE` - The name of the [MariaDB](https://mariadb.org/) database which defaults to `AssetDatabase`.
+* `MYSQL_USER` - The username used to connect to the MariaDB database which defaults to `assetManager`.
+* `MYSQL_PASSWORD` - The username used to connect to the [MariaDB](https://mariadb.org/) database which defaults to `assetManager`.
+* `s3Client__AccessKey` - The access key used to connect to the [S3](https://www.nuget.org/packages/Amazon.S3/) compliant object storage as described in the [Asset Manager documentation](https://github.com/ove/ove-asset-services/blob/master/packages/ove-asset-manager/README.md#configuration).
+* `s3Client__Secret` - The secret used to connect to the [S3](https://www.nuget.org/packages/Amazon.S3/) compliant object storage as described in the [Asset Manager documentation](https://github.com/ove/ove-asset-services/blob/master/packages/ove-asset-manager/README.md#configuration).
+* `s3Client__ServiceURL` - The service URL of the [S3](https://www.nuget.org/packages/Amazon.S3/) compliant object storage as described in the [Asset Manager documentation](https://github.com/ove/ove-asset-services/blob/master/packages/ove-asset-manager/README.md#configuration).
+* `ServiceHostUrl` - Hostname (or IP address) + port of the corresponding Asset Management Service.
+* `MariaDB__ConnectionString` - The connection string used by the [Asset Manager](https://github.com/ove/ove-asset-services/tree/master/packages/ove-asset-manager) to connect to [MariaDB](https://mariadb.org/) database. The format of the connection string must always be similar to what is provided, but the port number, database name, username and password must change accordingly if their default values were changed.
+* `AssetManagerHostUrl` - Hostname (or IP address) + port of the [Asset Manager](https://github.com/ove/ove-asset-services/tree/master/packages/ove-asset-manager).
+
+### Starting and stopping OVE
+
+OVE provides separate installers to help users install the necessary components. To install and start OVE on Docker run:
+
+```sh
+docker-compose -f docker-compose.ove.yml up -d
+```
+
+Replace the `-d` option above with `--no-start` if you want to install OVE but do not want to start it at the end of the installation process.
+
+Once the installation procedure has completed, the successful installation of OVE can be verified by accessing the [OVE Core API Documentation](http://OVE_CORE_HOST:PORT/api-docs) using a web browser.
```

Comment:
```suggestion
Once the installation procedure has completed, the successful installation of OVE can be verified by accessing the OVE Core API Documentation (located at `http://OVE_CORE_HOST:PORT/api-docs) using a web browser.
```

It is better to write out the URL: clicking the link won't work because the hostname and port number need to be replaced